tools/validators/instance_validator/build/lib/validate/entity_instance.py

```diff
@@ -0,0 +1,402 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Uses ontology universe to validate parsed instance data."""
```

Comment:
can you please use the original file structure (without build/lib) and put all the files in it.
It will requires us to update internal config on g3/3rd party to take into account the new path for tests.

would entity_instance_parser be helpful to distinguish between the parser and the logic?

Reply:
Oops, I think I accidentally committed this generated file in addition to the source file. As you noted, the original is also in this PR.

Reply:
now it makes sense :)